cirras-underwriting-war/cirras-underwriting-war/src/main/angular/src/app/components/inventory/forage/forage-inventory.component.ts

```diff
@@ -513,13 +505,34 @@ export class ForageInventoryComponent extends BaseComponent implements OnChanges
     }
   }
 
-  clearPlantInsurability(fieldIndex, plantingIndex, invSeededIndex) {
+  checkPlantInsurability(fieldIndex, plantingIndex, invSeededIndex) {
     const flds: FormArray = this.viewModel.formGroup.controls.fields as FormArray;
     const pltg = flds.controls[fieldIndex]['controls']['plantings'].value.controls[plantingIndex];
     const invSeeded = pltg.controls['inventorySeededForages'].value.controls[invSeededIndex];
 
-    invSeeded.controls['plantInsurabilityTypeCode'].setValue(null) 
-    this.isMyFormDirty()
```

Comment:
Do we still need the this.isMyFormDirty() check? I didn't see it in the updated version

Reply:
isMyFormDirty() was moved to the html part of the component, line 288. See changed files